dev/flights-cdh/sparkClusterDemo-source.R

```diff
@@ -0,0 +1,132 @@
+
+library(sparklyr)
+library(dplyr)
+library(ggplot2)
+
+Sys.setenv(HADOOP_CONF_DIR='/etc/hadoop/conf.cloudera.hdfs')
+Sys.setenv(YARN_CONF_DIR='/etc/hadoop/conf.cloudera.yarn')
+Sys.setenv(SPARK_HOME="/home/ubuntu/spark-1.6.0")
+Sys.setenv(SPARK_HOME_VERSION="1.6.0")
+
+sc <- spark_connect(master = "yarn-client", version="1.6.0", spark_home = '/opt/cloudera/parcels/CDH/lib/spark/')
```

Comment:
That's what I thought too, but if I leave out line 8, then the sparklyr job fails saying that it cannot find certain files in the Spark parcel folder